tracing-opentelemetry-kafka/build.gradle

```
@@ -9,7 +9,7 @@ dependencies {
     api platform (libs.boms.opentelemetry.instrumentation.alpha)
     api projects.micronautTracingOpentelemetry
     api libs.opentelemetry.api
-    api mnKafka.micronaut.kafka
+    compileOnly mnKafka.micronaut.kafka
```

I am not sure about this one.if You use `micronaut-tracing-telemetry-kafka` would not you want to get Micronaut Kafka automatically. 

```suggestion
    api(mnKafka.micronaut.kafka)
```